src/react-guides/Guides.tsx

```diff
@@ -135,12 +136,33 @@ export default class Guides extends React.PureComponent<GuidesProps, GuidesState
         style={style}
       >
         <div className={prefix('guide-origin')} ref={ref(this, 'originElement')} />
-        <Ruler ref={ref(this, 'ruler')} style={rulerStyle} {...this.rulerProps} />
+        <Ruler ref={ref(this, 'ruler')} style={this._rulerStyle} {...this.rulerProps} />
         {this.renderGuidesElements()};
       </GuidesElement>
     );
   }
 
+  private get _rulerStyle() {
+    return this.props.type === 'horizontal' ? {
+          ...this.props.rulerStyle,
+          marginLeft: `${this._offset}px`,
+          width: `calc(100% - ${this._offset}px)`,
+          height: '100%',
+          borderRight: 'none',
+          borderBottomWidth: '0.5px',
+          borderBottomStyle: 'solid',
+          borderBottomColor: this.props.theme === 'dark' ? DARK_THEME.borderColor : LIGHT_THEME.borderColor,
```

Comment:
I don't like the way how we change the theme.
I would like to have a method to check the current theme and add this to the variable like `currentTheme`.
In such case, we avoid using the ternary operator all time when we need to get some data.

example:
```
private getTheme() {
  this.currentTheme = this.props.theme === 'dark' ? DARK_THEME : LIGHT_THEME;
  return this.currentTheme;
}
```

and after you can use
```
borderBottomColor: this.currentTheme.borderColor
```